airbyte-config/config-models/src/test/java/io/airbyte/config/helpers/StateMessageHelperTest.java

```diff
@@ -22,13 +23,13 @@ public class StateMessageHelperTest {
 
   @Test
   public void testEmpty() {
-    final Optional<StateWrapper> stateWrapper = StateMessageHelper.getTypedState(null);
+    final Optional<StateWrapper> stateWrapper = StateMessageHelper.getTypedState(null, true);
```

Comment:
nit: could we define constants in this test like `USE_STREAM_CAPABLE_STATE = true` and `DONT_USE_STREAM_CAPABALE_STATE = false`, and use those in each of the calls to `getTypedState()`? I think that would make the test cases slightly more clear in what they are testing